packages/mgt-chat/src/components/ChatListItem/ChatListItem.spec.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { SampleChats, SampleMessagePreviewFromDexter } from './sampleData';
+import { test, expect } from '@playwright/experimental-ct-react17';
+import { ChatListItem } from './ChatListItem';
+
+test.describe('ChatListItem', () => {
+  test.use({ viewport: { width: 500, height: 500 } });
+
+  const chatSelected = { chatSelected: () => {} };
+
+  // group chat
+  // - member added
+  // - member removed
+  // - "You: " prefix if you sent the latest message
+  // - Member prefix if someone else sent the latest message
+  // - title is the topic
+  // - no topic, comma separated list of names
+  test('groupchat: Other persons message', async ({ mount }) => {
+    const component = await mount(
+      <ChatListItem chat={SampleChats.group.SampleGroupChat} myId="" onSelected={chatSelected} />
+    );
+    await expect(component).toContainText('Matthew: Hi everyone, call me Matt.');
+  });
+
+  test('groupchat: "You: " prefix on your message', async ({ mount }) => {
+    var chat = SampleChats.group.SampleGroupChat;
+    chat.lastMessagePreview = SampleMessagePreviewFromDexter;
+
+    const component = await mount(
+      <ChatListItem
+        chat={SampleChats.group.SampleGroupChat}
+        myId="a7530e0a-85f0-4f77-a617-2a5ab2a959ec"
+        onSelected={chatSelected}
+      />
+    );
+    component.textContent;
+    await expect(component).toContainText('You: Note to self');
+  });
+
+  test('groupchat: Member added message', async ({ mount }) => {
+    const component = await mount(
+      <ChatListItem chat={SampleChats.group.SampleGroupChatMembershipChange} myId="" onSelected={chatSelected} />
+    );
+    await expect(component).toContainText('Group Chat With New MembersDexter added Brandy');
+  });
+
+  test('groupchat: implicit topic as a list of names', async ({ mount }) => {
+    const component = await mount(
+      <ChatListItem chat={SampleChats.group.SampleGroupChat} myId="" onSelected={chatSelected} />
```

Comment:
I wasn't able to get Playwright to run successfully locally, but I think some of these tests are going to fail now due to the short-circuit being done in `ChatListItem` when `myId` is falsy.

Reply:
This was resolved in the latest commit.